packages/fether-ui/src/TokenCard/TokenCard.js

```diff
@@ -15,15 +15,23 @@ export const TokenCard = ({
   decimals,
   showBalance,
   token,
+  defaultTokenImage,
   ...otherProps
 }) => (
   <Card {...otherProps}>
     <div className='token'>
       <div className='token_icon'>
-        {token && token.logo ? (
-          <img alt={token.symbol} src={token.logo} />
+        {!!token && !!token.logo ? (
+          <img
+            alt={token.symbol}
+            src={token.logo}
+            onError={ev => {
```

Comment:
could you create a function called `handleError` or something like that? And do `onError={handleError}`.

Reason: small react performance optimization.

Reply:
As discussed, taking the function out of `TokenCard` prevent me from using `defaultTokenImage`.